package.json

```diff
@@ -11,6 +11,8 @@
     "babel-loader": "6.2.7",
     "babel-preset-react-app": "^2.0.1",
     "case-sensitive-paths-webpack-plugin": "1.1.4",
+    "chai-as-promised": "^7.1.1",
+    "chai-bignumber": "^2.0.2",
     "chalk": "1.1.3",
     "connect-history-api-fallback": "1.3.0",
     "cross-spawn": "4.0.2",
```

Comment:
Please add
`"ethereumjs-abi": "^0.6.5",`
to the package.json - your choice as to whether in `dependencies` or `devDependencies`. Would be inclined to go with the second if I were to have my way seeing as it is only required in the test suite.

Reply:
Ah, ok, I'll completely remove that, it isn't necessary in any of the tests yet, if it becomes necessary then I'll add it again.

Reply:
perfect